coreos-base/update_engine/update_engine-9999.ebuild

```diff
@@ -8,7 +8,7 @@ CROS_WORKON_REPO="https://github.com"
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="748eb99b18b67747f2e047327e9f2a18aa09d3b5" # flatcar-master
+	CROS_WORKON_COMMIT="b7504e70eef86663ba05208e162cbed8ef0d5712" # krnowak/glibc-2.34-ld
```

Comment:
Not sure if that's a correct commit.
It should be https://github.com/flatcar/update_engine/commit/1c290db26e5591c5fe3497d91040e186e12b4315 .

Reply:
Done.

Reply:
Thanks. Looks good.